008-infinispan-getting-started/src/test/java/org/acme/quickstart/containers/InfinispanTestResource.java

```diff
@@ -20,11 +21,17 @@
     @Override
     public Map<String, String> start() {
 
-        infinispan = new GenericContainer<>("infinispan/server:11.0.4.Final-2")
+        infinispan = new GenericContainer<>("infinispan/server:11.0.4.Final")
                 .waitingFor(new LogMessageWaitStrategy().withRegEx(".*Infinispan Server.*started in.*\\s"))
                 .withStartupTimeout(Duration.ofMillis(20000))
-                .withEnv("USER", "my_username")
-                .withEnv("PASS", "my_password");
+                .withClasspathResourceMapping("identities.yaml",
```

Comment:
Just curious - where did you find these config options?

It would be nice to add a comment with link to the config details page.

Reply:
Gonna add it to README